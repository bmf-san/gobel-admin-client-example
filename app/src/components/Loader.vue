<template>
  <div class="loader">Loading...</div>
</template>

<script>
export default {
  name: "Loader"
};
</script>