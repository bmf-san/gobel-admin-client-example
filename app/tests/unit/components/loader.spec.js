import { shallowMount } from "@vue/test-utils";
import Loader from "@/components/Loader.vue";

describe("Loader.vue", () => {
  it("init", () => {
    const wrapper = shallowMount(Loader);
    expect(wrapper.isVueInstance).toBeTruthy();
  });

  it("matched snapshot", () => {
    const wrapper = shallowMount(Loader);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
