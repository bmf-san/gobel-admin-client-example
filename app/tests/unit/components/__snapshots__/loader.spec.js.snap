// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.vue matched snapshot 1`] = `
<div class="loader">
  <div class="spinner">Loading...</div>
</div>
`;
